src/values/ObjectValue.js

```diff
@@ -242,7 +242,7 @@ export default class ObjectValue extends ConcreteValue {
   hashValue: void | number;
 
   // ReactElement
-  $BailOut: void | string;
+  $BailOut: void | string | AbstractValue;
```

Comment:
`string` is not okay, for a tracked property the full type should be `void | StringValue | AbstractValue`.

Reply:
Or, since you don't don't support merging of `$BailOut` properties anyway...

1. Revert the type to just `void | string`, and remove the code that rejects `AbstractValue`.
2. Remove it from the tracked list.
3. Where you set `$BailOut` message, consider the case where `$BailOut` has already been set, and then either merge the string messages manually, or produce some other kind of error, or consciously do something else (stick to first message, or always overwrite) with a comment as to what you do and why.